### PR TITLE
login 

### PR DESCRIPTION
**Pull request by NamrataRaikwar2002**
login credential: 
email: demo@gmail.com
password: demo

-login part 
  
csb: https://tie72x.csb.app/

### CODE REVIEW

File: src/Component/Drawer/Drawer.jsx [hunk 1]
```diff
@@ -1,13 +1,15 @@
 import './Drawer.css'
-export const Drawer = () => {
+import { Link } from 'react-router-dom'
+export const Drawer = ({sideBar}) => {
   return (
-    <>
-      <div className="drawerDiv page">
+     <div className={`drawerDiv page ${sideBar ? "displayBlock" : 'displayNone' }`}>
         <ul className="drawerUl">
+            <Link to='/'>
```
Comment (Kevin-Solomon) [inline src/Component/Drawer/Drawer.jsx:7]: Try using the ```NavLink``` component, so that we can use custom styles when the user is on the page.

**Review by Kevin-Solomon — APPROVED** [review]

File: src/Component/Drawer/Drawer.jsx (hunk 1 above)
Comment (NamrataRaikwar2002) [reply src/Component/Drawer/Drawer.jsx:7]: okk i will use it.

File: src/Pages/Authentication/Authentication.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+.passwordInputDiv{
+    border: none;
+}
+
+.login_page{
+    display: flex;
+    /* justify-content: center; */
+    /* border: 1px solid; */
```
Comment (kart-c) [inline src/Pages/Authentication/Authentication.css:59]: you can remove uncommented code if it is not required

**Review by kart-c — APPROVED** [review]
Apart from a small refactor to remove unnecessary comments in code, everything else looks good

**Pull request merged**